package.json

```diff
@@ -29,7 +29,7 @@
     "build:lib": "vite build",
     "build:lib:watch": "vite build --watch",
     "lint": "eslint .",
-    "prettier": "prettier . --check",
+    "format": "prettier . --check",
```

Comment:
Suggestion: I'd typically expect a `format` command to actually change my code rather than just report formatting errors, that way I don't have to remember the command to actually fix the formatting errors!

I'd suggest changing this to `prettier --write .`

Reply:
Good point. I've created a separate command for checking the format [e2853a5](https://github.com/guardian/interactive-component-library/pull/216/commits/e2853a5417dcf197e0d310adf79fee1e3d559560)